lib/jarvis/commands/teamtime.rb

```diff
@@ -0,0 +1,12 @@
+require "clamp"
+require "jarvis/location_times"
+
+module Jarvis module Command class Teamtime < Clamp::Command
+  banner "List the current time in the team locations"
+
+  def execute
+    STDERR.puts "sdfsdf"
+    puts Jarvis::LocationTimes.new.to_a.join($/)
```

Comment:
Being that this is for Jarvis, and it isn't entirely important, but we could use a mustache template for format the response, maybe.

We use mustache for Github comment templating: https://github.com/elastic/jarvis/blob/master/lib/jarvis/commands/merge.rb#L102-L105

I certainly won't require reworking this to use templates ;P